visidata/loaders/graphviz.py

```diff
@@ -20,7 +20,8 @@ def save_dot(vd, p, vs):
     srccol = vs.keyCols[0]
     dstcol = vs.keyCols[1]
     with p.open_text(mode='w', encoding='utf-8') as fp:
-        print('graph { concentrate=true;', file=fp)
+        print_wrap = print.__wrapped__
```

Comment:
Let's make this `p = lambda *args: print.__wrapped__(*args, file=fp)` and then adjust the following code accordingly.

Reply:
I am going to use `pfp` instead of `p` because `p` is the **Path** variable up-top!